Features5/tests/features/deleteContact.feature

```diff
@@ -0,0 +1,43 @@
+Feature: Delete Contact
+     As a user
+     I want to delete contact
+     So that I can free up the space
+    
+    Scenario Outline: User wants to delete particular contact
+        Given user has navigated to contact lists in the homepage containing following contacts:
+        |name    |email    |phone     |contactType   |
+        |<name>  |<email>  |<phone>   |<contactType> |
+
+        When user clicks "delete" button of following contact:
```

Comment:
```suggestion
        When user deletes the following contact with email:
        |email|
        |<email>|
```

we can find the delete button from email i guess so no need to pass all the data